quiche/src/h3/mod.rs

```diff
@@ -565,6 +569,35 @@ impl Config {
             self.connect_protocol_enabled = None;
         }
     }
+
+    /// Sets H3 additional settings.
+    ///
+    /// The default value is no additional settings.
+    /// The `additional_settings` parameter must not contain settings already
+    /// handled by default in this library.
```

Comment:
We should just list those settings here using the IANA names e.g. SETTINGS_QPACK_MAX_TABLE_CAPACITY, etc.

However, this raises an interesting future compatibility problem, where an application used additional settings prior to the library adding first-class support for them. Raising an error at runtime is sort of annoying in that situation, I wonder if there is some elegant way to make this a compiler error.

Reply:
Done in commit [d67f4ff](https://github.com/cloudflare/quiche/pull/1555/commits/d67f4ff5390a63a45ecfdec59b68241aebf29665).